internal/command/commands/env.go

```diff
@@ -291,6 +292,51 @@ func (e *Env) Run(args []string) int {
 		return 1
 	}
 
+	if e.Update {
```

Comment:
Add command helper for update check code line 365

Reply:
I have already added it